### PR TITLE
Add support to datetime on gt. gte. lt. lte. filter

### PR DESCRIPTION
**Pull request by cmoscosoz**
## What kind of change does this PR introduce?

Add support to datetime on "gt." supabase filter

## What is the current behavior?

throws an error: NoSuchMethodError: The method '>' was called on null.

## What is the new behavior?

accepts datetime value type and int value type

## Additional context

none

### CODE REVIEW

**Comment by dshukertjr** [comment]
Thanks for the amazing work! Would you have bandwidth to add the same feature on lt, gte, and lte as well?

**Comment by cmoscosoz** [comment]
@dshukertjr hey mate, no problem. leave it to me.

cheers,

**Comment by dshukertjr** [comment]
Legend! Feel free to bundle all of them in a single PR too 👍

**Comment by cmoscosoz** [comment]
> Legend! Feel free to bundle all of them in a single PR too 👍

@dshukertjr  ready for you review. cheers,

**Review by dshukertjr — APPROVED** [review]
This is awesome! Thanks for the amazing work!

**Comment by dshukertjr** [comment]
Will release a new version with this update right after I merge it!

**Pull request merged**
